docs/URLSchemeConfiguring.md

```diff
@@ -58,6 +58,17 @@ private fun handleLinkIntent(intent: Intent?) {
 
 ## Verifying the Custom URL Scheme
 
-Test that the URL scheme handling is set up correctly by navigating to `appcues-APPCUES_APPLICATION_ID://debugger` in your browser on the device with the app installed.
+The Appcues debugger allows you to easily validate that the Appcues deeplink is properly configured.
+
+1. Launch the debugger in your app with a call to ``Appcues/debug()``.
+2. Expand the debugger by tapping the floating button.
+3. Tap the "Appcues Deeplink Configured" row to verify the status. If a checkmark appears, the Appcues deeplink is properly configured.
+
+### Troubleshooting
```

Comment:
technically the Android SDK only gives error 1 or 2, if I recall.  Maybe we can remove the others if they do not apply here.